include/kimera_vio_ros/RosDataProviderInterface.hpp

```diff
@@ -10,30 +10,47 @@
 #include <functional>
 #include <string>
 
-#include <opencv2/core/core.hpp>
-#include <opencv2/core/matx.hpp>
+#include "opencv2/core/core.hpp"
+#include "opencv2/core/matx.hpp"
```

Comment:
We use <> for third-party includes

Reply:
I was having some issues with cmake prioritizing headers from the current opencv underlay workspace, given kimera is currently not compatible with the system's installed opencv headers, but perhaps I resolved this with f8184d26f173e2cb1bf036f21fe06e90838dfa3e . I'll test with a revert locally.